compiler/test/runnable/aligndefault.d

```diff
@@ -0,0 +1,28 @@
+
+struct S
+{
+    align(1)
+    {
+        short x1;
+        int y1;
+        long z1;
+
+        align(default)
+        {
+            short x;
+            int y;
+            long z;
+        }
+    }
+}
+
+void main()
+{
+    static assert(S.x1.alignof == 1);
```

Comment:
if this test contains only static asserts, then it should be a `compilable` test and this function should not be called main.